use_cases/eluc/prescriptors/heuristics/heuristics.py

```diff
@@ -0,0 +1,127 @@
+"""
+Heuristic to compare our prescriptors to.
+"""
+from abc import ABC, abstractmethod
+
+import pandas as pd
+
+from data import constants
+from predictors.predictor import Predictor
+from prescriptors.prescriptor import Prescriptor
+
+class HeuristicPrescriptor(Prescriptor, ABC):
+    """
+    Abstract heuristic prescriptor class that inherits from prescriptor class.
+    Has a percentage threshold that the heuristic is to reach but not exceed.
+    Also takes a predictor so that we can evaluate metrics.
+    Requires an implementation of reco_heuristic which takes a context dataframe and returns
+    recommendations based on the heuristic.
+    """
+    def __init__(self, predictor: Predictor):
+        self.predictor = predictor
+
+    @abstractmethod
+    def _reco_heuristic(self, pct: float, context_df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Abstract method that takes a percentage threshold of land change and a 
+        context dataframe and returns a dataframe of recommendations based on the heuristic.
+        """
+        raise NotImplementedError
+    
+    def prescribe_land_use(self, pct, __, context_df: pd.DataFrame) -> pd.DataFrame:
+        """
+        TODO: The header of this function is weird. Not entirely sure what to do about it.
+        The header of this function is kinda gacky. Not sure what to do about this...
+        We don't need a result dir or candidate id to load a prescriptor because we 
+        prescribe via. heuristic.
```

Comment:
Not entirely sure how to deal with this function header. I want it to match the other Prescriptors where you input: some parameter that changes how we prescribe and a context df. Maybe make id and results_dir one parameter in the Unileaf and PyTorch prescriptors.